stats-dashboard.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: blank
+title: Let's Encrypt Stats
+permalink: /stats-dashboard/
+---
+
+<div class="dashboard">
+
```

Comment:
Please add an HTML comment describing why this is here and linking to moz-corsica, so future maintainers know there is a dependency.